doc/stdlib/libc.rst

```diff
@@ -40,7 +40,10 @@
 
 .. function:: time(t::TmStruct)
 
-   Converts a ``TmStruct`` struct to a number of seconds since the epoch.
```

Comment:
Yes, that seems like a reasonable way to share docstrings. I definitely agree that we need a way to document bindings themselves as opposed to just objects – that's something I have an implementation in mind for and I just need some time to play around with it.